README.md

```diff
@@ -20,33 +22,33 @@ var randomCat = require('random-cat');
 // Random cat picture of various widths and heights
 // Possible values: 50, 100, 150, 200, 250, 300, 350, 400, 450, 500, 550, 600
 // http://lorempixel.com/100/600/cats
-var url = randomCat.get();
+var url = new randomCat.get();
```

Comment:
This is why I want to leave the `new RandomCat()`, to avoid all these new objects.  If we were to keep it without the `new RandomCat()`, rather than what you've done here, it'd be more appropriate to do this:

```
const randomCat = new RandomCat();
const url = randomCat.get();
```

Reply:
fine.